charts/s1-agent/values.yaml

```diff
@@ -39,6 +39,11 @@ configuration:
       communicator_enabled: true
     injection:
       enabled: false # to enable agent injection mode on AWS Fargate, set to 'true'
+    admissionControl:
```

Comment:
should it be admission_controller?
should it be under the helper section?
maybe "validating_admission_controller: false" ?
anyway we should verify the name with Yaniv since it is customer facing

Reply:
should it be admission_controller? I think it is semantic but I changed anyway
should it be under the helper section? I don't think so, it is like agent injection, it is not helper configuration although we use it to make the admission
maybe "validating_admission_controller: false" ? ok